openlibrary/templates/site.html

```diff
@@ -1,11 +1,15 @@
 $def with (page)
 
-$if not ctx.get("sent_head"):
+$if False and objhas(page, 'homepage') is True:
```

Comment:
These changes should have been rolled back and moved to refactor/slick-carousels